From 42d191fe179c9e74c15eb92d1a054ce2e319fb55 Mon Sep 17 00:00:00 2001
From: Jan Engelmohr <jan.engelmohr@mailbox.tu-dresden.de>
Date: Sun, 11 Mar 2018 13:57:04 +0100
Subject: [PATCH 1/2] kltechnduo: Fix RIL on encrypted devices

Using "start" makes the second ril daemon try to come up immediately which
fails because data isn't mounted yet.
So just enable the disabled service and let the class trigger do it's job.

Change-Id: I2fbc43a20c62c165d3467824cb878747abb7dfd2
---
 radio/rild_dsds.rc | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/radio/rild_dsds.rc b/radio/rild_dsds.rc
index 2f0a1b1..5c3381c 100644
--- a/radio/rild_dsds.rc
+++ b/radio/rild_dsds.rc
@@ -21,8 +21,8 @@ service ril-daemon2-gsm /vendor/bin/hw/rild -l /vendor/lib/libsec-ril-dsds.06w.s
     group radio cache inet misc audio log readproc wakelock oem_2901
     capabilities BLOCK_SUSPEND NET_ADMIN NET_RAW
 
-on main && property:rild.lib2_type=cdma
-    start ril-daemon2-cdma
+on property:rild.lib2_type=cdma
+    enable ril-daemon2-cdma
 
-on main && property:rild.lib2_type=gsm
-    start ril-daemon2-gsm
+on property:rild.lib2_type=gsm
+    enable ril-daemon2-gsm
-- 
2.15.1

